ipinfo/handler_utils.py

```diff
@@ -90,6 +91,20 @@ def read_country_names(countries_file=None):
 
     return json.loads(countries_json)
 
+def read_eu_country_names(eu_countries_file=None):
```

Comment:
I think we can just use the code above in `read_country_names` but make it more generic. It basically just opens a file and parses it as JSON, and that's it, no schema-specific stuff in Python since it's all dynamic.